pkg/util/k8s/k8s_test.go

```diff
@@ -2,6 +2,7 @@ package k8s
 
 import (
 	"context"
+	"github.com/libopenstorage/operator/pkg/constants"
```

Comment:
move this import in the group below. ideally we should maintain 3 groups: golang internal, external portworx vendors, everything else